wallet-crypto/src/redeem.rs

```diff
@@ -141,6 +141,8 @@ impl Signature {
         let bytes = hex::decode(hex)?;
         Self::from_slice(&bytes)
     }
+
+    pub fn to_bytes<'a>(&'a self) -> &'a [u8;SIGNATURE_SIZE] { &self.0 }
```

Comment:
:+1:

Reply:
This got removed in the following commit